fees/pangolin/index.ts

```diff
@@ -0,0 +1,44 @@
+import { CHAIN } from "../../helpers/chains";
+import { getUniV2LogAdapter } from "../../helpers/uniswap";
+import { Adapter } from "../../adapters/types";
+
+const adapter: Adapter = {
+  version: 2,
+  adapter: {
+    [CHAIN.AVAX]: {
+        fetch: getUniV2LogAdapter({
+          factory: "0xefa94DE7a4656D787667C749f7E1223D71E9FD88",
+          fees: 0.003,
+          customLogic: ({ dailyVolume, dailyFees }) => {
+            // Protocol Revenue (Treasury): 0.0075% out of 0.3% swap fees
+            const dailyRevenue = dailyFees.clone(0.0075 / 0.3);
```

Comment:
I think this should be dailyProtocolRevenue,

dailyRevenue is protocol revenue + holder revenue 
![image](https://github.com/user-attachments/assets/72d0d59f-53f2-46ea-af9d-762f0f8302b9)